docs/bots-guide.md

```diff
@@ -0,0 +1,188 @@
+# Developing bots
+**This feature is still experimental**
+Bots are a new system in Zulip, allowing developers to react to messages.
+
+With bots, you *can*
+
+* intercept and view messages sent by users on Zulip
+* send out new messages
+
+With bots, you *cannot*
+
+* modify an intercepted message (you have to send a new message)
+* send messages in the name of other users
+* intercept private messages
+
+
+On this page you'll find:
+
+* A step-by-step [tutorial](#how-to-deploy-a-bot) on how to deploy a bot.
+* A step-by-step [tutorial](#how-to-develop-a-bot) on how to develop a bot.
+* A [documentation](#bot-api) of the bot API.
+* Common [problems](#common-problems) when developing/deploying bots and their solutions.
+
+Contributions to this guide are very welcome, so if you run into any
+issues following these instructions or come up with any tips or tools
+that help writing integration, please email
+zulip-devel@googlegroups.com, open an issue, or submit a pull request
```

Comment:
I just noticed that I didn't change the "writing integration" part, will do so as well.